Gemfile

```diff
@@ -34,7 +34,7 @@ group :production do
   gem 'twitter'
   gem 'hipchat'
   gem 'flowdock'
-  gem 'json'
+  gem 'json', '2.0.2'
```

Comment:
You shouldn't need to hardcode a specific version of json any more.  The json gem 1.8.6 and 2.0.3 both support all versions of ruby.  2.0.3 now only requires ruby >=1.9, and 1.8.6 supports ruby >=2.0.